app/controllers/admins_controller.rb

```diff
@@ -19,4 +23,20 @@ def create
     end
   end
 
+  def disable
+    merchant = Merchant.find(params[:id])
+    merchant.items.each {|item| item.update_attribute(:active?, false)}
+    merchant.update_attribute(:active?, false)
+    redirect_to "/admin/merchants"
+    flash[:notice] = "disabled #{merchant.name}"
+  end
+
+  def enable
+    merchant = Merchant.find(params[:id])
+    merchant.items.each {|item| item.update_attribute(:active?, true)}
+    merchant.update_attribute(:active?, true)
+    redirect_to "/admin/merchants"
+    flash[:notice] = "enabled #{merchant.name}"
+  end
```

Comment:
I bet there is a way to refactor and eliminate the repetition! Maybe add it to the column on the project board to come back to if there is time?

Reply:
Yup! I'm thinking we'll be able to use a partial here. Already added it to the board ;)

Reply:
Would love to help with that refactor since I dont fully grasp partials.

Reply:
I've never implemented one before and am still fuzzy on the exacts of how to do it. I'd love to go through it with anyone else who wants to too

Reply:
I think that partials has to do with the view so I'm not 100% sure how it will help here